base_layer/core/src/chain_storage/db_transaction.rs

```diff
@@ -202,6 +202,9 @@ impl DbTransaction {
     /// Insert a "chained" orphan block.
     /// The transaction will rollback and write will return an error if the orphan already exists.
     pub fn insert_chained_orphan(&mut self, orphan: Arc<ChainBlock>) -> &mut Self {
+        if orphan.block().body.inputs().iter().any(|b| b.is_compact()) {
+            panic!("Compact inputs are not supported in the blockchain");
```

Comment:
Panic seems wrong here

Reply:
Whoops good catch, that was just an extra check for me while testing that I forgot to take out (co-pilot wrote the text :))